backend/LexBoxApi/Services/PermissionService.cs

```diff
@@ -141,6 +141,34 @@ public async ValueTask AssertCanManageProjectMemberRole(Guid projectId, Guid use
             throw new UnauthorizedAccessException("Not allowed to change own project role.");
     }
 
+    public async ValueTask<bool> CanCreateGuestUserInProject(Guid? projectId)
```

Comment:
I suspect these don't need to be nullable anymore

Reply:
You're right. Fixed in commit bd2425b9. Then I'll resolve this conversation so I can merge the PR.